tasks/task_finminer.yaml

```diff
@@ -0,0 +1,18 @@
+container:
+  commit_on_stop: false
+  image: sonm/finminer:latest
+  env:
+    # Required    
+    WALLET: ""
+    # (Optional) Coin to mine. Supported values: ETH (by default), ETC, XMR, ETN, PASC. These have higher priority than the "algorithm" parameter. An arbitrary coin can also be specified.
+    COIN: ""
```

Comment:
Comment all non-required parameters

Reply:
run.sh include checks for empty value for optional parameters

Reply:
Sure, but in other examples, we're keeping optional parameters as comments. Just for consistency. :)